CSS/style.css

```diff
@@ -0,0 +1,40 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.app {
+    
+    height: 100vw;
+    width: 100vw;
```

Comment:
Applying your `.app` div a `width: 100vw` is unnecessary and might cause unwanted glitches here. Parent of the `.app` div is `body` tag and the default width of the `body` element in CSS is `width: auto` which means that its width will expand to fill the width of its container, which is typically the browser viewport. So if you remove `width: 100vw` .app will strech 100% of it's parent (`body`).